src/SourceBuild/content/eng/sdkArchiveDiffs.targets

```
@@ -0,0 +1,55 @@
+<Project>
+
+  <UsingTask AssemblyFile="$(SdkArchiveDiffTasksAssembly)" TaskName="GetValidArchiveItems" />
+  <UsingTask AssemblyFile="$(SdkArchiveDiffTasksAssembly)" TaskName="GetClosestOfficialSdk" />
+  <UsingTask AssemblyFile="$(SdkArchiveDiffTasksAssembly)" TaskName="FindArchiveDiffs" />
+
+  <Target Name="ReportSdkArchiveDiffs"
+          AfterTargets="Build"
+          DependsOnTargets="DetermineSourceBuiltSdkVersion">
+
+      <Message Text="Comparing built SDK against closest official build"
+               Importance="High"/>
+
+      <GetValidArchiveItems ArchiveItems="@(SdkTarballItem)"
+                            ArchiveName="dotnet-sdk">
+        <Output TaskParameter="ValidArchiveItems"
+                ItemName="_BuiltSdkArchivePath"/>
+      </GetValidArchiveItems>
+
+      <!-- There should only be 1 SDK archive -->
+      <Error Text="Multiple valid dotnet-sdk archives found."
+             Condition="'@(_BuiltSdkArchivePath->Count())' != '1'" />
+
+      <GetClosestOfficialSdk BuiltArchivePath="@(_BuiltSdkArchivePath)">
+        <Output TaskParameter="ClosestOfficialArchivePath"
+                PropertyName="_ClosestOfficialSdkPath" />
+      </GetClosestOfficialSdk>
+
+      <FindArchiveDiffs BaselineArchive="@(_BuiltSdkArchivePath)"
+                        TestArchive="$(_ClosestOfficialSdkPath)">
+        <Output TaskParameter="ContentDifferences"
+                ItemName="_ContentDifferences" />
+      </FindArchiveDiffs>
+
+      <ItemGroup>
+        <_changedFiles Include="@(_ContentDifferences)" Condition="'%(_contentDifferences.Kind)' != 'Unchanged'" />
+      </ItemGroup>
+
+      <PropertyGroup>
+        <SdkArchiveDiffsReport>$(ArtifactsLogDir)\SdkArchiveDiffs\SdkArchiveDiffs.csv</SdkArchiveDiffsReport>
```

I don't think we need a sub-folder:

```suggestion
        <SdkArchiveDiffsReport>$(ArtifactsLogDir)SdkArchiveDiffs.csv</SdkArchiveDiffsReport>
```